### DISCUSSION TITLE
"Missing shortcut" Error after building for windows

### DISCUSSION THREAD

**Discussion by byOppes** [alex8088/electron-vite#357] [Q&A]
Hello! I'm new to Electron development, and haven't been able to figure out this problem when I build my app for distribution.
I don't get any errors in development nor production preview environments, and when running the `npm run build:win` command I didn't get any errors either.

I'm able to run the installer from _myApp/dist/myApp-1.0.0-setup.exe_ without any problems, but as soon as the installation is completed, the desktop shortcut appears without the proper icon and when I try to launch my app I get an error saying that it is missing a shortcut. When I choose to locate the shortcut I am presented with "_..Appdata/Local/Programs/myApp_" folder that only has the uninstaller file in it.

Anyone have any clue where I might have gone wrong? Apologies if I have not provided enough info, but as said: I'm new to this and don't really know which files to show or discuss.
(Note: I used myApp name here just for the sake of simplicity)

Package.json:
<img width="627" alt="packageJSON" src="https://github.com/alex8088/electron-vite/assets/78928366/4731dbc5-8974-4e76-a955-0669e5c9cb59">

electron-builder.yml:
<img width="882" alt="electron-builder" src="https://github.com/alex8088/electron-vite/assets/78928366/30db1672-f412-47d1-b70c-6aed48e1cb41">

**Comment by charlesanim** [comment]
@byOppes Any updates here ? Were you able to resolve this issue ? I'm running into this issue as well and was hoping to find a resolution. @alex8088 thoughts ?
